libs/data-access/adverts/src/lib/saga.ts

```diff
@@ -64,9 +64,11 @@ export function* getAdvertById(actions) {
     const id = actions.payload;
     const path = `/api/${PrivateRoutesPath.ADVERTS}/${id}`;
     const result = yield call(get, path);
-    if (result) {
+
+    if (!!result) {
```

Comment:
in this case double exclamation mark isn't necessary, because if result returns something it be already true. This case necessary if in react component you checking some case with && operator